lightly/api/version_checking.py

```diff
@@ -0,0 +1,60 @@
+from typing import Tuple
+
+import requests
+
+from lightly.openapi_generated.swagger_client import VersioningApi, VersionNumber
+from lightly.openapi_generated.swagger_client.api_client import ApiClient
+
+from lightly.openapi_generated.swagger_client.configuration import Configuration
+from lightly.api.utils import getenv
+
+
+def get_versioning_api() -> VersioningApi:
+    configuration = Configuration()
+    configuration.host = getenv('LIGHTLY_SERVER_LOCATION', 'https://api.lightly.ai')
+    token = getenv('TOKEN', None)
+    configuration.api_key = {'token': token}
+    api_client = ApiClient(configuration=configuration)
+    versioning_api = VersioningApi(api_client)
+    return versioning_api
+
+
+def get_latest_version(current_version: str) -> Tuple[None, str]:
+    try:
+        versioning_api = get_versioning_api()
+        version_number: str = versioning_api.get_latest_pip_version(current_version = current_version)
+        return version_number
+    except Exception as e:
+        return None
+
+
+def get_minimum_compatible_version():
+    versioning_api = get_versioning_api()
+    version_number: str = versioning_api.get_minimum_compatible_pip_version()
+    return version_number
+
+
+def version_compare(v0, v1):
+    v0 = [int(n) for n in v0.split('.')][::-1]
+    v1 = [int(n) for n in v1.split('.')][::-1]
```

Comment:
This works for the moment but maybe we'll have to find a better way to do this...